index.html

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portifolio</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body> <!-- Header start-->
+    <header>
+        <div class="nav-container">
+            <a href="#" class="logo">WELCOME</a>
+            <nav>
+                <button class="menu" type="button" >
+                    <img src="images/menu.gif" alt="a menu link">
+                </button>
+            </nav>
+        </div>
+    </header>   <!-- Header end-->
+
+     <!-- main start-->
+    <section class="main">
+        <div class="intro">
+            <h1>Hey there. I’m Yordi ! </h1>
+            <h1><span class="a-span">I’m a software developer</span></h1>
+        </div>
+        <div class="header-info">
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="media-link">
+            <ul class="socials">
+                <li><a href="#"><img src="images/twitter.png" alt="twitter image"></a></li>
+                <li><a href="#"><img src="images/Linkedin icon.png" alt="linkedin image"></a></li>
+                <li><a href="#"><img src="images/medium.png" alt="medium image"></a></li>
```

Comment:
- [ ] Kindly fix the background image however, remember to implement it with CSS and not through the HTML file because the background is for styling the page.
### The figma design
![image](https://user-images.githubusercontent.com/86200356/198714596-d92266c1-e2e2-42bc-a7f8-0dcef76d68fd.png)

### Your design
![image](https://user-images.githubusercontent.com/86200356/198714833-42e1e7aa-5efc-475e-9d43-3e9bdeebea9a.png)